docroot/modules/humsci/hs_blocks/hs_blocks.install

```diff
@@ -78,3 +78,21 @@ function _hs_blocks_fix_sections(array $sections) {
   }
   return $was_changed;
 }
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function hs_blocks_update_dependencies() {
+  // Permissions for field_paragraph_style need to be removed first.
+  $dependencies['hs_blocks'][10201] = [
+    'su_humsci_profile' => 9714,
+  ];
+  return $dependencies;
+}
```

Comment:
This hook doesn't appear necessary. The `su_humsci_profile_update_9714` has already been deployed to production environment and therefore will never be run again. This hook is only needed to identify the order of update hooks, but since it's already been ran, the dependency is irrelevant.
If the two update hooks were deploying at the same time, maybe. But also since this is all just a mono-repo it would be better to just update the other hook if that were the case. And even so, adding and removing permissions in two different hooks shouldn't cause any conflicts. they run independently of each other. The order is irrelevant.

Reply:
You're right. I was needed when the PR was initially created, but not anymore.